frame/session/benchmarking/Cargo.toml

```diff
@@ -26,6 +26,7 @@ rand = { version = "0.7.2", default-features = false }
 [dev-dependencies]
 serde = { version = "1.0.126" }
 codec = { package = "parity-scale-codec", version = "2.0.0", features = ["derive"] }
+scale-info = "0.10.0"
```

Comment:
Is this actually being used by the benchmarks?

Reply:
Yes because of the mock runtime, which adds `TypeInfo` derives to the generated types. This requires a direct `scale-info` dependency, same as for `codec`